assets/src/application/ui/layout/entityList/filterBar/buttons/FilterButtonWrap.tsx

```diff
@@ -15,4 +15,5 @@ const FilterButtonWrap: React.FC<FilterButtonWrapProps> = ({ children, id, isMob
 	</div>
 );
 
-export default withViewportMatch({ isMobile: '< small' })(FilterButtonWrap);
+export default FilterButtonWrap;
+// export default withViewportMatch({ isMobile: '< small' })(FilterButtonWrap);
```

Comment:
`withViewportMatch` violated React Hook Rules.

Reply:
after this is merged to master i'll try to solve that problem using CSS